src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1444,7 +1444,14 @@ public static void DCS_GenericQueue()
     {
         Queue<int> value = new Queue<int>();
         value.Enqueue(1);
-        var deserializedValue = SerializeAndDeserialize<Queue<int>>(value, "<QueueOfint xmlns=\"http://schemas.datacontract.org/2004/07/System.Collections.Generic\" xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\"><_array xmlns:a=\"http://schemas.microsoft.com/2003/10/Serialization/Arrays\"><a:int>1</a:int><a:int>0</a:int><a:int>0</a:int><a:int>0</a:int></_array><_head>0</_head><_size>1</_size><_tail>1</_tail><_version>2</_version></QueueOfint>");
+        object syncRoot = ((ICollection)value).SyncRoot;
```

Comment:
What's the purpose of accessing SyncRoot... to lazily-initialize it, and if so, does that affect the serialization?  If so, do you need a test for when it hasn't been initialized?